ws-service/types.ts

```diff
@@ -1,8 +1,22 @@
 import { TransactionWithAddressAndPrices } from 'services/transactionService'
 
 type TxBroadcast = 'NewTx' | 'OldTx'
+
 export interface BroadcastTxData {
   address: string
   txs: TransactionWithAddressAndPrices[]
   messageType: TxBroadcast
 }
+export interface BroadcastTransactionPayload {
+  address: string
+  transactions: Transaction[]
+  messageType: TxBroadcast
+}
```

Comment:
I suggested changing `txs` to `transactions` because it is clearer to understand what it refers to. But, I get that we use `tx` and `txs` in many places in the code, which could create ambiguity, but also, I noticed that `transaction` is also used, so I thought it would be acceptable to make the change.

the second object `BroadcastTransactionPayload` differs from the first not only because `txs` was renamed, but also because it points to another `Transaction` type that is more simplified, as suggested by the issue. I kept the old `BroadcastTxData` there because it is used in some other logic and I focused on updating the data being sent to the client only

Reply:
thanks for reviewing, yeah, I do think `BroadcastTransactionPayload` and `BroadcastTxData` are equal in some sense, but there is a difference because it is a payload so it is kinda referring to its use for send it in some function as a payload and not other logic. 

But indeed is confusing so I was planning to do a clean up and change it to `BroadcastTransactionResponse` or something more clear as you suggested. 

about sticking to `tx` for me it is okay, it was just a suggestion because I particularly think that when we use this codes in the code it can be harder to read, but in this case it is minimum and we are used to `tx` so let's continue using `tx`

Reply:
About naming the broadcasted transaction referring to its specific use in the BroadcastTransactionPayload, sure, I think this is a good thing.